nym-wallet/src/pages/balance/components/vesting-timeline.tsx

```diff
@@ -30,18 +34,30 @@ export const VestingTimeline: React.FC<{ percentageComplete: number }> = ({ perc
     <Box display="flex" flexDirection="column" gap={1} position="relative" width="100%">
       <svg width="100%" height="12">
         <rect y="2" width="100%" height="6" rx="0" fill="#E6E6E6" />
-        <rect y="2" width={`${percentageComplete}%`} height="6" rx="0" fill="#121726" />
+        <rect
+          y="2"
+          width={`${percentageComplete}%`}
+          height="6"
+          rx="0"
+          fill={mode === 'light' ? '#121726' : '#21D073'}
+        />
         {vestingAccountInfo?.periods.map((period, i, arr) => (
           <Marker
             position={`${calculateMarkerPosition(arr.length, i)}%`}
-            color={+percentageComplete.toFixed(2) >= calculateMarkerPosition(arr.length, i) ? '#121726' : '#B9B9B9'}
+            color={
+              +percentageComplete.toFixed(2) >= calculateMarkerPosition(arr.length, i)
+                ? mode === 'light'
+                  ? '#121726'
+                  : '#21D073'
```

Comment:
Hi, these color values exist in the wallet palette, `text.dark` for `#121726`, `success` for `#21D073`, should be better to access them from the theme instead of using hard coded values.